src/components/Log/AddPopOver.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <ion-content class="ion-padding">
+    <ion-item 
+      @click="$router.push({ name: 'AddCatalog' })"
+      button
+    >
+      <h5 class="title-tag">from catalog</h5>
```

Comment:
Standardize your name or at minimum capitalize it please!

Reply:
like "From Catalog"?